content/en/Platform Deep Dive/Scans/_index.md

```diff
@@ -14,16 +14,16 @@ The complexity of today's applications, combined with the constant evolution of
 
 Scans address these challenges head-on. By combining the power of PtaaS with DAST, you gain a comprehensive solution for continuously assessing risk.
 
-The beta period runs until December 31, 2023. During the beta, you have access to three scans a month, allowing you to fully explore this new feature. Scans reset on the first of the month. Anticipate usage and limitation adjustments in 2024.
-
-To access the Scans Beta you must be an org owner or org member. Staff org members can also access this section to be able to better provide support.
+To get started with Scans, you will need to add a target. Every Cobalt Customer has access to one target for free unlimited scanning. Additional targets can be purchased. Learn more about [targets](/platform-deep-dive/scans/targets).
 
 We use the following IP Address to make requests to your target: **52.19.40.38**
 
 ## How to Use Scans
 
 [Watch the how-to video here.](https://videos.cobalt.io/watch/4D8kQcTmPBE3HdWy6X3fHW)
 
+### Add a Target
```

Comment:
Should this go into `targets.md` ?

Reply:
I think we can just link the Targets page within the Scans how-to, so that we don't break up the instructions.